packages/rocketchat-action-links/server/actionLinkHandler.js

```diff
@@ -0,0 +1,58 @@
+//Action Links Handler. This method will be called off the client.
+
+Meteor.methods({
+	actionLinkHandler(fuctIndex, msgObj, userIdCalled) {
+		var rcDataBaseObj = RocketChat.models.Messages.findOne({ _id: msgObj._id });
+		// console.log(rcDataBaseObj);
+		
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2673573741/issues/source?bid=3199634&fileBranchId=3333125#l7)